web-app/src/modules/core/routes/Validators/components/ValidatorsStats.tsx

```diff
@@ -23,8 +59,47 @@ const ValidatorsStats: FC<ValidatorsStatsProps> = ({ validators }) => {
       <StatsCard title="Liquid Libra" value={liquidLibra ? liquidLibra : null}>
         {liquidLibra && <Money>{liquidLibra}</Money>}
       </StatsCard>
+      <StatsCard title="Vouch Price" value={utils ? utils.vouchPrice : null}>
+        {utils && <Money>{Math.ceil(utils.vouchPrice / 1_000_000)}</Money>}
+      </StatsCard>
+      <StatsCard title="Entry Fee" value={utils ? utils.entryFee : null}>
+        {utils && <Money>{Math.ceil(utils.entryFee / 1_000_000)}</Money>}
+        <span className="ml-2" style={{ fontSize: '16px' }}>
+          ({utils && formatPercentage(utils.clearingBid)})
+        </span>
+      </StatsCard>
+      <StatsCard title="Net Reward" value={utils ? utils.netReward : null}>
+        {utils && <Money>{Math.ceil(utils.netReward / 1_000_000)}</Money>}
+      </StatsCard>
+      {/* <StatsCard title={thermostatTitle} value={utils ? utils.vouchPrice : null}>
+        {utils && <Money>{Math.ceil(utils.thermostatMeasure.amount / 1_000_000)}</Money>}{' '}
+        <span style={{ fontSize: '16px' }}>
+          (
+          {utils &&
+            formatPercentage(
+              utils.thermostatMeasure.percentage,
+              utils.thermostatMeasure.amount !== 0,
+              utils.thermostatMeasure.didIncrease,
+            )}
+          )
+        </span>
+      </StatsCard>*/}
     </div>
   );
 };
 
+// format percentage 1 decimal
+function formatPercentage(value: number) {
+  console.log('value', value);
+  return `${(value / 10).toFixed(1)}%`;
+}
+
+// print percentage 0-100
```

Comment:
Is this code intentionally commented out? If so why?

Reply:
Same as above. Thanks for pointing out.